test/contconic.jl

```diff
@@ -503,5 +503,106 @@ function contconictest(solver::MOI.AbstractSolver, ε=Base.rtoldefault(Float64))
         end
     end
 
+    if MOI.supportsproblem(solver, MOI.ScalarAffineFunction, [(MOI.ScalarAffineFunction{Int}, EqualTo{Int}), (MOI.ScalarAffineFunction{Int}, EqualTo{Rational{Int}}), (MOI.VectorVariablewiseFunction, MOI.PositiveSemidefiniteConeTriangle), (MOI.VectorVariablewiseFunction, MOI.SecondOrderCone)])
```

Comment:
It's not ok to return a `ConstraintReference{ScalarAffineFunction{Float64},LessThan{Float64}}` if the user provided a `(ScalarAffineFunction{Int}, LessThan{Int})` constraint.